datumaro/tests/test_coco_format.py

```diff
@@ -39,18 +38,20 @@ def __iter__(self):
                                 ), label=0,
                                 id=2, group=2, attributes={'is_crowd': True}),
                         ]
-                    ),
-                ])
+                    )]
 
-            def categories(self):
-                label_cat = LabelCategories()
-                label_cat.add('TEST')
-                return { AnnotationType.label: label_cat }
+        # Creating categories
+        label_cat = LabelCategories()
+        label_cat.add('TEST')
+        categories = { AnnotationType.label : label_cat}
+
+        # Creating exemplary dataset
+        exemplary_dataset = Dataset.from_iterable(items_list, categories)
 
         dataset = Project.import_from(DUMMY_DATASET_DIR, 'coco') \
             .make_dataset()
 
-        compare_datasets(self, DstExtractor(), dataset)
+        compare_datasets(self, exemplary_dataset, dataset)
```

Comment:
I suggest to stick to the naming used everywhere else with `source`, `target` or `actual` and `expected`.

Reply:
In general, tests follow AAA-pattern https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80